hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/listeners/DomainEventListener.kt

```diff
@@ -52,7 +45,7 @@ class DomainEventListener(
       when (eventType) {
         in incentiveEvent -> indexListenerService.incentiveChange(fromJson(message), eventType)
         in restrictedPatientEvent -> indexListenerService.restrictedPatientChange(fromJson(message), eventType)
-        in alertEvent -> indexListenerService.alertChange(fromJson(message), eventType)
+        "person.alerts.changed" -> indexListenerService.alertChange(fromJson(message), eventType)
```

Comment:
This is more effeceint, and also fixes the Nomis batch problem with an alert being made inactive